main.cpp

```diff
@@ -303,19 +314,20 @@ void Piano::arpeggio(int keyNum, int range, int interval)
 //copied UDT 3:
 struct WaveGenerator
 {
-    int mOscillator = 1;
+    int oscillator = 1;
```

Comment:
`oscillator` is a strange name for something holding an integer.
What is this variable actually storing? 
The wave type? 
Name things what they are.
- [ ] revise